src/uct/ib/rc/base/rc_iface.c

```diff
@@ -289,14 +293,17 @@ ucs_status_t uct_rc_iface_flush(uct_iface_h tl_iface, unsigned flags,
     }
 
     count = 0;
+    ucs_spin_lock(&iface->eps_lock);
```

Comment:
no deadlock danger here that ep_flush will try to take some lock?

Reply:
`uct_ep_flush` below does not have UCT completion, am i missing something?

Reply:
for now I see ep flush would not take spinlock, so i guess it's ok, better than lock for every ep